src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -123,6 +126,24 @@ class AggregatorSuite extends SparkSuite {
     p.check()
   }
 
+  @Test def testMaxMin() {
+    val rdd = sc.parallelize(Seq(
+      Annotation("a",  0,    1, -1.0, null,  1,   -1, null, null),
```

Comment:
Why use 1.0 and 1f?  Seems like 1-4 are the same as 5-8.  Why not just compute the min and max of 1-4?